license-excludes.xml

```diff
@@ -0,0 +1,6 @@
+<excludes>
+  <exclude>src/.circleci/circleci-readme.md</exclude>
+  <exclude>src/.github/**</exclude>
+  <exclude>src/Makefile</exclude>
+  <exclude>src/Dockerfile.goreleaser</exclude>
+</excludes>
```

Comment:
No newline, (lol this is totally me, making a comment to not forget it)

Reply:
> No newline, (lol this is totally me, making a comment to not forget it)

Done anyways. 9a18429